### ISSUE TITLE
Unbox installs to ./npmbox.cache and not ./node_modules

### ISSUE THREAD

**Issue by TimothyWrightSoftware** [arei/npmbox#56]
I am not sure if I'm doing something wrong, but some modules do not seem to work.  
I have installed the npmunbox on my offline machine.  If I do:

`npmunbox --save require.npmbox` then require and all the dependencies are placed into the node_modules and the package.json is updated.

if I try the same thing with webpack,
`npmunbox --save webpack` or `npmunbox --save-dev webpack` with or without the .npmbox extension, the command line says "Unpacking.." and then finishes.  All the data seems to be going into the .npm-cache folder but then nothing else happens.

I pulled npmbox 26 July 2016, and i'm using Ubuntu 16.04

**Comment by avrahamcool** [comment]
I had the same issue whith 'http-server' (Unpacking, but not installing) while other packages worked fine.

**Comment by arei** [comment]
Did you try the --verbose flag? What's the output? Can you paste it here?

**Comment by avrahamcool** [comment]
```
C:\Users\***\Desktop\New folder>npmunbox --verbose http-server

Unboxing http-server...
  Unpacking...
```

and thats it.

**Comment by TimothyWrightSoftware** [comment]
I did the same thing with verbose, (but linux) and same output.  I believe this is the same bug as the "Large packages don't work" problem with the archive library.

**Comment by TimothyWrightSoftware** [comment]
I tried setting the version back to 1.0.2 but the npmbox npmbox to install on the offline computer pulls the latest from the repo anyway.  I'm not an npm wizard, but I'm sure there's a workaround

**Comment by TimothyWrightSoftware** [comment]
I tried cloning the repo and changing the dependencies, but even though the right dependencies are pulled and installed for npmbox, when I run "npmbax npmbox" to create the offline installer, the changes to package.json are ignored.  I changed the repository url to point to my clone, and I checked in the changes, but it seems to pull from the master no matter what.

**Comment by Bucabug** [comment]
Having the same issue here but with pm2.

**Comment by arei** [comment]
Fixed in 4.0.0.
